src/DynamoCore/Graph/Workspaces/SerializationConverters.cs

```diff
@@ -334,22 +334,13 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
             serializer.Serialize(writer, ws.Nodes);
 
             //notes
-           writer.WritePropertyName("Notes");
+            writer.WritePropertyName("Notes");
```

Comment:
@ikeough - I am fine with that. Sid and I were actually looking for Notes in the Annotations block earlier this week during testing.